src/components/Component/Component.js

```diff
@@ -0,0 +1,13 @@
+import { Component } from "react";
+
+class ClassComponent extends Component {
+  render() {
+    return (
+      <div className="ClassComponent">
```

Comment:
if you don't have styles for these classnames, don't write them. And which methodology are using `ClassComponent`  type of class naming?

Reply:
I don't use any methodology,I just named as Component